lib/broccoli/angular2-app.js

```diff
@@ -31,28 +41,36 @@ Angular2App.prototype.toTree = function() {
       'angular2/bundles/upgrade.dev.js'
     ];
 
+
+
     if (this.options && this.options.vendorNpmFiles) {
       vendorNpmFiles = vendorNpmFiles.concat(this.options.vendorNpmFiles);
     }
 
     var tsConfigCompilerOptions = JSON.parse(fs.readFileSync('src/tsconfig.json', 'utf-8')).compilerOptions;
+    // TODO(i): kill rootFilePaths in broccoli-typescript and use tsconfig.json#files instead
+    tsConfigCompilerOptions.rootFilePaths = JSON.parse(fs.readFileSync('src/tsconfig.json', 'utf-8')).files;
```

Comment:
Note; `broccoli-typescript` might be going away with the upgrade to broccoli 1.0.